### DISCUSSION TITLE
gotMetadataFromURL: http error 0: No Response

### DISCUSSION THREAD

**Discussion by doreenhui** [transmission/transmission#2450] [Q&A]
I also encountered the same problem, when pulling a seed to download,and it works. but then when pulling another seed, will encounter such a problem, how to solve it?

**Comment by ikwyl6** [comment]
have same issue with transmission 3.0 only recently for the last week.
```
$ transmission-daemon -V
transmission-daemon 3.00 (bb6b5a062e)
```
happens when I copy the download link of a torrent and paste it into 'Open Torrent' popup in transmission web client. Has worked for me for long time until recently. If I alternatively - download the torrent file and then add it using the same method, the torrent gets added to my torrent list but doesn't download. It just sits in 'Idle' state.

**Comment by ikwyl6** [comment]
So I dug around some more and a few days ago I was trying to disable ip6 (unsuccessfully) and since I had "fe80::" in my bind-address-ipv6 setting, I believe this was causing me the issue. So I removed it so it stayed as the default of "::" and this has apparently fixed my problem..
